app/services/idv/gpo_mail.rb

```diff
@@ -17,11 +17,7 @@ def profile_too_old?
       current_user.pending_profile.created_at < min_creation_date
     end
 
-    # Next two methods are analytics helpers used from GpoController and ReviewController
-
-    # Caveat: If the user succeeds on their final phone attempt, the :proof_address
-    # RateLimiter is reset to 0. But they probably wouldn't be doing verify by mail
-    # if they succeeded on the phone step.
+    # Next two methods are analytics helpers used from GpoController and EnterPasswordController
```

Comment:
Prior rename
```suggestion
    # Next two methods are analytics helpers used from RequestLetterController and EnterPasswordController
```

Reply:
I addressed this in https://github.com/18F/identity-idp/pull/9378